style.css

```diff
@@ -0,0 +1,657 @@
+body {
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+nav {
+  display: flex;
+  color: #4f545d;
+  justify-content: space-between;
+  margin: 0px 6% 0px 6%;
+  align-items: center;
+  margin-top: 10px;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  margin-right: 50px;
+  padding-right: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;

```

Comment:
remove this and check once, any way parent has the same css properties

Reply:
resolved